modules/cnf-image-based-upgrade-share-container-directory.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-image-based-upgrade-shared-container-directory_{context}"]
+= Configuring a shared container directory between `ostree` stateroots
+
+You must apply a `MachineConfig` to both the seed and the target clusters during installation time to create a separate partition and share the `/var/lib/containers` directory between the two `ostree` stateroots that will be used during the upgrade process.
+
+[IMPORTANT]
+====
+You must complete this procedure at installation time.
+====
+
+.Procedure
+
+. Apply a `MachineConfig` to create a separate partition.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 98-var-lib-containers-partitioned
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+        - device: /dev/disk/by-id/wwn-<root_disk> <1>
```

Comment:
Can this be /dev/disk/by-path? Using path means the user can define the structure of this MC based solely off the type of hardware being deployed rather than requiring a unique value to be inserted here for every system deployed. (important for break-fix/redeployment on replacement hardware scenarios as well).
The rendered version also has an ignitionConfigOverride with the same consideration (don't see it in this PR).